### PR TITLE
init model

### PR DESCRIPTION
**Pull request by kienpham2004**
## Related Tickets:
-  [#32076](https://edu-redmine.sun-asterisk.vn/issues/32076)

## WHAT this PR does?
fix convention model

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by kienpham2004** [comment]
ready

File: app/Models/Comment.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    public function posts()
+    {
+        return $this->belongsTo(Post::class);
+    }
+
+    public function activities()
+    {
+        return $this->morphToMany(Activity::class,'actable');
```
Comment (huyenttm-0901) [inline app/Models/Comment.php]: convention

File: app/Models/Role.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+class Role extends Model
+{
+    protected $fillable = [
+        'name',
+    ];
+
+    public function users()
+    {
+        return $this->hasMany(User::class,'role_id','id');
```
Comment (huyenttm-0901) [inline app/Models/Role.php]: tương tự

**Review by huyenttm-0901 — COMMENTED** [review]
xem lại link redmine

**Comment by kienpham2004** [comment]
ready

**Pull request merged**